### PR TITLE
Update rate and Dead Address Balance check at kashiPair 

### PR DESCRIPTION
**Pull request by sarangparikh22**

### CODE REVIEW

**Comment by matthewlilley** [comment]
@sarangparikh22 @Clearwood burning 1 unit should be enough

File: contracts/KashiPair.sol [hunk 1]
```diff
@@ -144,7 +144,9 @@ contract KashiPair is ERC20, BoringOwnable, IMasterContract {
         require(address(collateral) == address(0), "KashiPair: already initialized");
         (collateral, asset, oracle, oracleData) = abi.decode(data, (IERC20, IERC20, IOracle, bytes));
         require(address(collateral) != address(0), "KashiPair: bad pair");
-
+        // to avoid share reset condition
+        require(bentoBox.balanceOf(asset, 0x000000000000000000000000000000000000dEaD) >= 1000, "KashiPair: dead balance less");
```
Comment (matthewlilley) [inline contracts/KashiPair.sol:148]: Burning 1 unit should be enough

**Review by matthewlilley — CHANGES_REQUESTED** [review]

**Comment by boringcrypto** [comment]
No need to burn 1000. 1 is enough. That will enforce there's always at least 1000 balance in the BentoBox.

File: contracts/KashiPair.sol (hunk 1 above)
Comment (sarangparikh22) [reply contracts/KashiPair.sol]: @matthewlilley updated

**Comment by sarangparikh22** [comment]
> No need to burn 1000. 1 is enough. That will enforce there's always at least 1000 balance in the BentoBox.

@boringcrypto updated

**Review by matthewlilley — APPROVED** [review]

**Pull request merged**